pyproject.toml

```diff
@@ -43,7 +43,7 @@ pyright = ">=1.1.396"
 poethepoet = ">=0.16.5"
 loguru = ">=0.6.0"
 typing-extensions = ">=4.4.0"
-matplotlib = ">=3.5.1,<3.9.0" # TODO https://github.com/pandas-dev/pandas/issues/58851
+matplotlib = ">=3.5.10" # TODO https://github.com/pandas-dev/pandas/issues/58851
```

Comment:
pandas now requires 3.6.3, so can update this constraint to that version, and remove the comment since that issue is now closed

Reply:
I will adjust this is a good point.